docs/install.md

```diff
@@ -246,3 +251,54 @@ information.
 We maintain [one-click apps and install scripts for cloud
 providers](https://github.com/cdr/deploy-code-server) such as DigitalOcean,
 Railway, Heroku, and Azure.
+
+## Uninstall
+
+Removing code-server depends on where you installed it.
+
+To delete settings and data:
+
+```shell
+rm -rf ~/.local/share/code-server ~/.config/code-server
+```
+
+### install.sh
+
+If you installed with the install script, by default code-server will be in `~/usr/lib/code-server` and you can remove it with `rm -rf`. e.g.
```

Comment:
❓ Is `~/usr/lib/code-server` the right path? 🤔

Reply:
According to `install.sh`
> The release is unarchived into ~/.local/lib/code-server-X.X.X
>      and the binary symlinked into ~/.local/bin/code-server

So no that is not right.

Reply:
Updated for the unarchived. Wondering if we also need to tell them to remove the binary in `/bin`